packages/client/src/pages/Forum/types.ts

```diff
@@ -0,0 +1,34 @@
+import { Dispatch, SetStateAction } from 'react';
+
+export interface newTopicProps {
+  setIsNewTopicOpen: Dispatch<SetStateAction<boolean>>;
+}
+
+export interface MessagesTypes {
+  data: {
+    author: string;
+    message: string;
+    time: string;
+    id: string;
+  }[];
+}
```

Comment:
Лишняя вложенность во всех типах в этом файле: `data { ... }`.  
Ее можно убрать. Например, вместо `MessagesTypes` экспортни отсюда тип  Message. Будет `useState<Message[]>`. Иметь отдельный тип для сообщения в единственном числе (а не для массива сообщений)удобнее для разных манипуляций. Он пригодится например в будущем компоненте `<Message />`

То же самое с остальными типами в этом файле

```suggestion
export interface Message {
  author: string;
  message: string;
  time: string;
  id: string;
}
```